packages/doltlab/content/introduction/getting-started.md

```diff
@@ -186,7 +186,7 @@ If `docker ps` works without `sudo`, you are in business.
 
 Now, my host is set up for DoltLab, I need to generate the configuration needed to run it. This is done with the installer as well, this time by only passing in the name of your host.
 
-```sh
+````sh
```

Comment:
four ticks not three?

Reply:
Ah it added that when I saved because this block was missing closing backticks. Just fixed